distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -12,6 +12,7 @@ The ${command-console} has a command-line option for ingesting data.
 [NOTE]
 ====
 Ingesting with the console ingest command creates a metacard in the catalog, but does not copy the resource to the content store.
+See the URL Resource Reader documentation for more information on retrieving the resource from a directory.
```

Comment:
@peterhuffer not sure if you think this is sufficient for the ingest command docs or if we want to update something with the actual command as well.

Reply:
I think putting this in a warning would be good and adding the link to the URL Resource Reader - <<_url_resource_reader, URL Resource Reader>>.